config.json

```
@@ -2464,6 +2464,26 @@
           "multiple-clause-matching"
         ],
         "difficulty": 5
+      },
+      {
+        "slug": "affine-cipher",
+        "name": "Affine Cipher",
+        "uuid": "18d12bac-ead0-456c-bcef-ccb2fd06fe72",
+        "prerequisites": [
+          "pattern-matching",
```

Pattern matching needs some friends to reach its full potential 😁 
```suggestion
          "pattern-matching",
          "guards",
          "multiple-clause-functions",
```